library/core/src/array/mod.rs

```diff
@@ -832,7 +832,7 @@ where
         }
     }
 
-    let mut array = MaybeUninit::uninit_array::<N>();
+    let mut array = [const { MaybeUninit::uninit() }; N];
```

Comment:
There should probably be an example for using inline const for this somewhere in the MaybeUninit docs.

Reply:
Done!